build.rs

```diff
@@ -12,6 +12,36 @@
 // WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+#![deny(
```

Comment:
No way to just have these in a "global" file somewhere, something like a `./clippy.toml`? :(

Reply:
I don't think it is available at the moment, the `clippy.toml` file seems to be for [configuring existing lints](https://github.com/rust-lang/rust-clippy#configuration).